Source/Classes/NantesLabel.swift

```diff
@@ -510,13 +523,15 @@ public extension NSAttributedString.Key {
     }
 
     override open func touchesEnded(_ touches: Set<UITouch>, with event: UIEvent?) {
-        guard let activeLink = activeLink else {
-            super.touchesEnded(touches, with: event)
+        if let activeLink = activeLink {
+            handleLinkTapped(activeLink)
+        } else if let touch = touches.first, forceTapsOntoAttributionToken && attributedTruncationToken(at: touch.location(in: self)) {
             labelTappedBlock?()
-            return
+        } else if let labelTappedBlock = labelTappedBlock {
```

Comment:
I think you need to add `, !forceTapsOntoAttributionToken` to here, otherwise taps outside the attributed token will still go through (as the above statement will be false, but this will be true)

Reply:
Yeah, good catch. I have to rewrite this once I change the boolean as well.